x-pack/plugins/apm/scripts/bootstrap_apm/README.md

```diff
@@ -0,0 +1,56 @@
+# Getting started
+
+A script for creating an elastic stack deployment on a remote cluster using `ecctl` and generating APM data using [apm-synthtrace](https://github.com/elastic/kibana/tree/main/packages/elastic-apm-synthtrace) tool.
+
+## Prerequisites for running the script
+
+- [ecctl](https://www.elastic.co/downloads/ecctl) installed
+- API key
+  - You can generate a key on [staging](https://staging.found.no/deployment-features/keys)
+- Make sure you have installed Kibana dependencies
+
+## Configuration
+
+Update the following configuration with your own credentials and store it in your home directory `$HOME/.ecctl/config.json`
+
+```
+$HOME/.ecctl/config.json
+
+{
+  "host": "https://staging.found.no",
+  "api_key": "API_KEY",
+  "region": "us-west-2",
+  "output": "text",
+  "timeout": 30000000000,
+  "insecure": true
+}
+```
+
+## Run script
+
+From kibana root
+
+`bash x-pack/plugins/apm/scripts/bootstrap_apm/src/run.sh -n 'apm-bootstrap-with-data'`
+
+The following options are supported:
+| Option | Description | Default |
+| ------------------| ------------------------------------------------------- | ------------ |
+| `-v` | Stack version | `7.17.0`
+| `-n` | Deployment name | `performance-apm-cluster`
+| `-r` | Region | `gcp-us-central1`
+| `-p` | Hardware profile | gcp-cpu-optimized`
+
+The script will try to bootstrap automatically a remote cluster with the following settings and it will use [01_simple_trace.ts](https://github.com/elastic/kibana/tree/main/packages/elastic-apm-synthtrace/src/scripts/examples/01_simple_trace.ts) to generate data
```

Comment:
In a follow up PR we should have a flag to select a scenario by name.

Reply:
Added two flags to configure apm-synthtrace(https://github.com/elastic/kibana/pull/123791/commits/41cfe8bc3af49271b4934f4f2ad1279eeecc7acc) 

1. `-s` for data scenario
2.  `-o` for synthtrace options 

For example, 
`bash x-pack/plugins/apm/scripts/bootstrap_apm/src/run.sh -v "7.16.3" -s 03_monitoring.ts -o '--workers 3 --clientWorkers 2'`